.github/workflows/php.yml

```diff
@@ -1,11 +1,17 @@
 name: PHP Linting and Tests
 
-on: pull_request
+on:
+    pull_request:
+        branches:
+            - feature/*
```

Comment:
We run them for all pull requests, not only for feature branches.
Same for JS.

Reply:
Fixed here d199e80fda803eaf427850b96361cdd783984a8d